test/integrations/deployers/DeployerCoordinator.t.sol

```diff
@@ -320,6 +320,85 @@ contract DeployerCoordinatorTest is HyperdriveTest {
         assertEq(deployment.target4, targets[3]);
     }
 
+    function test_deployTarget_frontrun() external {
+        // Deploy a target0 instance as deployer (should succeed).
+        coordinator.deployTarget(DEPLOYMENT_ID, config, new bytes(0), 0, SALT);
+
+        vm.stopPrank();
+        vm.startPrank(bob);
+        // Try to frontrun the target1 deployment with bob
```

Comment:
Nit: Let's keep comments at the top of the code block (in this case, we'd move it above `vm.stopPrank()`).